Makefile

```diff
@@ -19,10 +21,15 @@ endif
 KONG_PACKAGE_NAME?="kong-community-edition"
 KONG_CONFLICTS?="kong-enterprise-edition"
 KONG_LICENSE?="ASL 2.0"
-KONG_SOURCE_LOCATION?="$$PWD/../kong/"
-KONG_VERSION?="0.0.0"
 PRIVATE_REPOSITORY?=true
 KONG_TEST_CONTAINER_NAME?=localhost:5000/kong
+KONG_SOURCE_LOCATION?="$$PWD/../kong/"
+KONG_VERSION?=`echo $(KONG_SOURCE_LOCATION)/kong-*.rockspec | sed 's,.*/,,' | cut -d- -f2`
```

Comment:
Any chance this line and [this line](https://github.com/Kong/kong/blob/3a8abf8591e9f73228732150920e3cd18b2cce1b/Makefile#L22) could be DRYed up into a ENVVAR in the `.requirements` file?

Reply:
not at this time. Right now there's too much plumbing in other places that expect the rockspec to be versioned. Future us should try to collate to one source of truth 👍